src/Socket.php

```diff
@@ -281,7 +285,7 @@ public static function getSocketLinkTypes()
      *    - display
      * @return string ID of the select
      **/
-    public static function dropdownWiringSide($name, $options = [])
+    public static function dropdownWiringSide($name, $options = [], $full = false)
```

Comment:
```suggestion
    public static function dropdownWiringSide($name, $options = [], bool $full = false)
```

Also, please add param in PHPDoc.